WordPress/Classes/ViewRelated/Me/Me Main/VerifyEmailRow.swift

```diff
@@ -0,0 +1,230 @@
+import Foundation
+import UIKit
+import SwiftUI
+import WordPressUI
+import Combine
+
+final class VerifyEmailRow: ImmuTableRow {
+    static let cell = ImmuTableCell.class(VerifyEmailCell.self)
+    let action: ImmuTableAction? = nil
+
+    func configureCell(_ cell: UITableViewCell) {
+        // Do nothing.
+    }
+}
+
+final class VerifyEmailCell: UITableViewCell {
+    private let hostingView: UIHostingView<VerifyEmailView>
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        hostingView = .init(view: .init())
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupView() {
+        selectionStyle = .none
+        backgroundColor = .systemRed.withAlphaComponent(0.9)
+
+        contentView.addSubview(hostingView)
+        hostingView.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
```

Comment:
Please use extensions from `WordPressUI`:

```swift
contentView.addSubview(hostingView)
hostingView.pinEdges(to: contentView)
```

`hostingView.translatesAutoresizingMaskIntoConstraints` will not be needed if you do.